isamples_metadata/GEOMETransformer.py

```diff
@@ -30,10 +30,6 @@ def __init__(self, source_record: typing.Dict):
                     GEOMEChildTransformer(source_record, child_record)
                 )
 
```

Comment:
Since I was creating a new one of these, it seemed like a good time to revisit previous implementations and clean up mistakes.  All the changes in files other than Smithsonian are simple refactorings to clean the code up.